File: 32-email-tracking/python-sdk/email.py
import os, requests
API = "https://api.costkatana.com/api"
KEY = os.getenv("COST_KATANA_API_KEY")

def send_report(recipient, report_type):
    res = requests.post(f"{API}/email/report",
        json={"recipient": recipient, "reportType": report_type},
        headers={"Authorization": f"Bearer {KEY}"})
    res.raise_for_status()
    print(f"✅ Report sent to: {recipient}")
    return res.json()["data"]

File: 31-email-tracking/npm-package/email.ts
import axios from 'axios';
const API = 'https://api.costkatana.com/api'\;
const KEY = process.env.COST_KATANA_API_KEY;

export async function sendReport(recipient: string, reportType: string) {
  const res = await axios.post(`${API}/email/report`, {recipient, reportType},
    {headers: {'Authorization': `Bearer ${KEY}`}});
  console.log('✅ Report sent to:', recipient);
  return res.data.data;
}
